spotbugs/src/main/java/edu/umd/cs/findbugs/detect/FindDeadLocalStores.java

```diff
@@ -254,18 +262,41 @@ private void analyzeMethod(ClassContext classContext, Method method) throws Data
             BugInstance pendingBugReportAboutOverwrittenParameter = null;
             try {
                 WarningPropertySet<WarningProperty> propertySet = new WarningPropertySet<>();
+
```

Comment:
This method is quite long as it is, with this change, ranging from line 207 to 645. IMO it's worth considering moving some parts of it into separate functions. (Not necessarily in this PR.)